### ISSUE TITLE
 Inconsistent additionalPrinterColumns jsonPath

### ISSUE THREAD

**Issue by lburgazzoli** [fabric8io/kubernetes-client#4610]
### Describe the bug

I have a class defined like:

```java
@JsonInclude(JsonInclude.Include.NON_NULL)
@Buildable(builderPackage = "io.fabric8.kubernetes.api.builder")
public class DeploymentSpec {

    @PrinterColumn(name = "CONNECTOR_TYPE_ID")
    private String connectorTypeId;
}
```

And such class is used in bot the spec and the status of a resource.

### Fabric8 Kubernetes Client version

5.12.4

### Steps to reproduce

Generate CRDs

### Expected behavior

The generated CRD should always use the same path to describe the CONNECTOR_TYPE_ID column.
The resulted CRD may get amended at every build, i.e:

![image](https://user-images.githubusercontent.com/1868933/204271235-0add5674-e801-4f6d-b655-22135df24a25.png)

### Runtime

Kubernetes (vanilla)

### Kubernetes API Server version

1.25.3@latest

### Environment

Linux

### Additional context

```xml        
        <kubernetes-client.version>5.12.3</kubernetes-client.version>
        <java-operator-sdk.version>3.1.1</java-operator-sdk.version>
        <quarkus.version>2.11.2.Final</quarkus.version>
        <quarkus-operator-sdk.version>4.0.1</quarkus-operator-sdk.version>
        <sundrio.version>0.50.2</sundrio.version>
```

**Comment by metacosm** [comment]
Note that this probably won't get fixed for 5.12, only for 6.x as new 5.12.x releases are currently not planned.

**Comment by lburgazzoli** [comment]
> Note that this probably won't get fixed for 5.12, only for 6.x as new 5.12.x releases are currently not planned.

Not an issue for us
